deps/config/types.go

```diff
@@ -258,129 +282,138 @@ type CurioSubsystemsConfig struct {
+	// This step submits the generated proofs to the chain. (Default: false)
 	EnableUpdateSubmit bool
 
-	// UpdateEncodeMaxTasks sets the maximum number of concurrent SnapDeal encoding tasks that can run on this instance.
+	// UpdateEncodeMaxTasks sets the maximum number of concurrent SnapDeal encoding tasks that can run on this instance. (Default: 0 - unlimited)
 	UpdateEncodeMaxTasks int
 
-	// UpdateProveMaxTasks sets the maximum number of concurrent SnapDeal proving tasks that can run on this instance.
+	// UpdateProveMaxTasks sets the maximum number of concurrent SnapDeal proving tasks that can run on this instance. (Default: 0 - unlimited)
 	UpdateProveMaxTasks int
 
 	// EnableWebGui enables the web GUI on this curio instance. The UI has minimal local overhead, but it should
-	// only need to be run on a single machine in the cluster.
+	// only need to be run on a single machine in the cluster. (Default: false)
 	EnableWebGui bool
 
-	// The address that should listen for Web GUI requests.
+	// The address that should listen for Web GUI requests. It should be in form "x.x.x.x:1234" (Default: 0.0.0.0:4701)
 	GuiAddress string
 
 	// UseSyntheticPoRep enables the synthetic PoRep for all new sectors. When set to true, will reduce the amount of
-	// cache data held on disk after the completion of TreeRC task to 11GiB.
+	// cache data held on disk after the completion of TreeRC task to 11GiB. (Default: false)
 	UseSyntheticPoRep bool
 
 	// The maximum amount of SyntheticPoRep tasks that can run simultaneously. Note that the maximum number of tasks will
-	// also be bounded by resources available on the machine.
+	// also be bounded by resources available on the machine. (Default: 0 - unlimited)
 	SyntheticPoRepMaxTasks int
 
-	// Batch Seal
+	// EnableBatchSeal enabled SupraSeal batch sealing on the node.  (Default: false)
 	EnableBatchSeal bool
 
-	// EnableDealMarket enabled the deal market on the node. This would also enable libp2p on the node, if configured.
+	// EnableDealMarket enabled the deal market on the node. This would also enable libp2p on the node, if configured. (Default: false)
 	EnableDealMarket bool
 
 	// EnableCommP enables the commP task on te node. CommP is calculated before sending PublishDealMessage for a Mk12 deal
-	// Must have EnableDealMarket = True
+	// Must have EnableDealMarket = True (Default: false)
 	EnableCommP bool
 
 	// The maximum amount of CommP tasks that can run simultaneously. Note that the maximum number of tasks will
-	// also be bounded by resources available on the machine.
+	// also be bounded by resources available on the machine. (Default: 0 - unlimited)
 	CommPMaxTasks int
 
 	// The maximum amount of indexing and IPNI tasks that can run simultaneously. Note that the maximum number of tasks will
-	// also be bounded by resources available on the machine.
+	// also be bounded by resources available on the machine. (Default: 8 - unlimited)
```

Comment:
Nit: this reads like "8 is unlimited, also the default"

Reply:
My bad. Will fix this.